src/sentry/incidents/tasks.py

```diff
@@ -16,23 +19,27 @@
     IncidentStatusMethod,
 )
 from sentry.models import Project
-from sentry.services.hybrid_cloud.user import user_service
+from sentry.services.hybrid_cloud.user import RpcUser, user_service
 from sentry.snuba.dataset import Dataset
+from sentry.snuba.models import QuerySubscription
 from sentry.snuba.query_subscription_consumer import register_subscriber
 from sentry.tasks.base import instrumented_task
 from sentry.utils import metrics
 from sentry.utils.email import MessageBuilder
 from sentry.utils.http import absolute_uri
 
+if TYPE_CHECKING:
```

Comment:
Is it needed? Why not pull SubscriptionUpdate in here

Reply:
Oh maybe not, I was trying to fix a circular import issue but that was not actually a fix. Working on it now.